firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionInitiator.kt

```diff
@@ -28,23 +30,23 @@ import android.os.Bundle
  * @hide
  */
 internal class SessionInitiator(
-  private val currentTimeMs: () -> Long,
+  private val elapsedRealtime: () -> Duration,
```

Comment:
Could you add a comment to explain why this is elapsed time instead of current time (eg. what the difference between this and the iOS SDK)?

Reply:
It looks like this is just time since boot instead of unix timestamt for ios

Reply:
When playing with the clocks for this PR, I found that unix time is not a good way to measure intervals like session length. Unix time can change when the customer changes the time on their device, daylight savings, or other funny timezone things even leap seconds, etc. Maybe iOS should change to a different clock, something like this that is a monotonic uniform clock.